cocos/network/SocketIO.cpp

```diff
@@ -116,10 +430,10 @@ SIOClientImpl::~SIOClientImpl()
 
 void SIOClientImpl::handshake()
 {
-    log("SIOClientImpl::handshake() called");
+    CCLOGINFO("SIOClientImpl::handshake() called");
 
     std::stringstream pre;
-    pre << "http://" << _uri << "/socket.io/1";
+    pre << "http://" << _uri << "/socket.io/1/?EIO=2&transport=polling&b64=true";
```

Comment:
I just re verified, there was no problem, even if the url doesn't exist, the handshake can pass. What we met was another problem, so please ignore it